yolact.py

```diff
@@ -396,18 +44,31 @@ class Yolact(nn.Module):
         - pred_aspect_ratios: A list of lists of aspect ratios with len(selected_layers) (see PredictionModule)
     """
 
-    def __init__(self):
+    def __init__(self, 
+                 config_name="yolact_base_config",
+                 ):
+        """
+        @param config_name: string name of used config, choose from ./data/config.py, default "yolact_base"
+        """
         super().__init__()
 
+        #set (custom) config
+        ## FIXME set_cfg does not work? For now set directly
+        from data.config import set_cfg
```

Comment:
outdated, this actually does work

Reply:
Any reason why this is imported here and not in the preamble?